store-implementation/simple-federated-store/src/main/java/uk/gov/gchq/gaffer/federated/simple/FederatedStore.java

```diff
@@ -150,20 +161,34 @@ public void removeGraph(final String graphId) {
      * @throws IllegalArgumentException If graph not found.
      */
     public GraphSerialisable getGraph(final String graphId) throws CacheOperationException {
-        GraphSerialisable graph = graphCache.getFromCache(graphId);
-        if (graph == null) {
+        return getGraphAccessPair(graphId).getLeft();
```

Comment:
would be good to also have a helper method for getRight so we don't have to know the pair ordering in the operation handlers